RELEASE.md

```diff
@@ -117,6 +119,8 @@ $ sed -i "s/v1.0.0/v1.1.1/g" ./README.md
 4. Send a pull request with the changes. In the description, explain the steps to verify the hash update, i.e., reviewers shoud LGTM only if the provenance verification succeeds
 and the hash in the pull request matches the one computed on the binary. You can use [#slsa-framework/slsa-github-generator#113](https://github.com/slsa-framework/slsa-github-generator/pull/113) as example.
 
+5. Update the e2e.installer-action.yml [here](https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.installer-action.yml#L9) and [here](https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.installer-action.yml#L17) with the newly released version; and [here](https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.installer-action.yml#L27) with its corresponding commit sha.
```

Comment:
nit: linking to specific lines is pretty fragile. Maybe update with instructions like "replace all version references to the slsa-verifier repo with references to the newly released version".

OR maybe add a comment in the `e2e.installer-action.yml` like `# UPDATE ON RELEASE` for each of the locations that need to be updated.

Reply:
done, thanks